tests/baselines/reference/decoratorOnClass8.errors.txt

```diff
@@ -6,6 +7,7 @@ tests/cases/conformance/decorators/class/decoratorOnClass8.ts(3,1): error TS2322
     
     @dec()
     ~~~~~~
-!!! error TS2322: Type '(target: Function, paramIndex: number) => void' is not assignable to type '(target: typeof C) => void | typeof C'.
+!!! error TS1220: Invalid expression for class decorator.
+!!! error TS1220:   Supplied parameters do not match any signature of call target.

```

Comment:
Yeah, that would be a lot clearer - perhaps we could do the same for tagged templates, but let's both as a separate PR to keep this review manageable.